src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double base1;
+    private double base2;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double base1, double base2, double height) {
+        super(color);
+        this.base1 = base1;
+        this.base2 = base2;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return ((base1 + base2) / 2) * height;
+    }
+
+    @Override
+    protected String getName() {
+        return "isosceles trapezoid";

```

Comment:
Class names should be nouns and not contain verbs. Consider renaming the class to something like `IsoscelesTrapezoidFigure` to better reflect its nature as an entity.